framework/core/js/src/admin/components/ExtensionsWidget.tsx

```diff
@@ -1,12 +1,19 @@
-import app from '../../admin/app';
-import DashboardWidget from './DashboardWidget';
+import app from '../app';
+import DashboardWidget, {IDashboardWidgetAttrs} from './DashboardWidget';
 import isExtensionEnabled from '../utils/isExtensionEnabled';
 import getCategorizedExtensions from '../utils/getCategorizedExtensions';
 import Link from '../../common/components/Link';
 import icon from '../../common/helpers/icon';
+import { Extension } from '../AdminApplication';
+import type Mithril from 'mithril';
```

Comment:
Let's import types as only types 👍🏼 

```suggestion
import DashboardWidget from './DashboardWidget';
import isExtensionEnabled from '../utils/isExtensionEnabled';
import getCategorizedExtensions from '../utils/getCategorizedExtensions';
import Link from '../../common/components/Link';
import icon from '../../common/helpers/icon';
import type { Extension } from '../AdminApplication';
import type { IDashboardWidgetAttrs } from './DashboardWidget';
import type Mithril from 'mithril';
```